src/services/services.ts

```diff
@@ -6092,13 +6195,32 @@ namespace ts {
                 findInStrings: boolean,
                 findInComments: boolean,
                 result: ReferencedSymbol[],
-                symbolToIndex: number[]): void {
+                symbolToIndex: number[],
+                indexToSymbol: {[index: number]: Symbol}): void {

```

Comment:
this does not seem to be used anywhere except for recording the symbol, is it still needed?